applications/mlflow/chart/mlflow/templates/mlflow-waitfor-secret.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.mlflow.trackingServer.basicAuth.enabled }}
+{{- if not .Values.mlflow.trackingServer.basicAuth.existingSecret }}
+apiVersion: v1
+stringData:
+  POSTGRES_USER: {{ .Values.postgres.auth.username }}
+  POSTGRES_PASSWORD: {{ .Values.postgres.auth.password }}
+  {{- if $.Values.postgres.external.enabled }}
+  POSTGRES_HOST: {{ .Values.postgres.external.host }}
+  POSTGRES_PORT: {{ .Values.postgres.external.port }}
+  POSTGRES_DB: {{ .Values.postgres.external.database }}
+  {{- else }}
+  POSTGRES_HOST: {{ printf "%s-postgres-rw" (include "mlflow.fullname" .) }}
+  POSTGRES_PORT: "5432"
+  POSTGRES_DB: {{ .Values.postgres.embedded.initdb.database }}
+  {{- end }}
+kind: Secret
+metadata:
+  name: {{ printf "%s-waitfor-postgres" (include "mlflow.fullname" .)  | trunc 63 | trimAll "-" }}
+type: Opaque
```

Comment:
Any reason these are at the end? It's just strange to see it written this way even though I know it's fine.

Reply:
no reason, but i don't like it either :)